WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -1647,20 +1660,27 @@ nick_the_quarreller = {
 nick_the_devil = {
 	allow = {
 		ruled_years = 10
-		trait = wroth
+		OR = {
+			trait = wroth
+			trait = cannibal_trait
+			has_character_modifier = cannibal
+		}
```

Comment:
But this is a general part nickname about evilness, it isn't "the Evil Cannibal".